### PR TITLE
German translation update + update.sh change

### PR DESCRIPTION
**Pull request by berberic2**
- updated german translation and

- added option to run update.sh for selected languages
  p.e. `update.sh de`  only updates german translation

### CODE REVIEW

File: translations/update.sh
Comment (hasecilu) [inline translations/update.sh:1]: Could you add and example in case someone doesn't know how to use that?
Usage: ./update.sh de

**Comment by shaise** [comment]
Thanks!

File: translations/update.sh
Comment (berberic2) [reply translations/update.sh:1]: I’m not sure what you mean. An usage for `--help` like in:
https://github.com/berberic2/FreeCAD_FastenersWB/tree/update.sh

Comment (hasecilu) [reply translations/update.sh:1]: Hi, I was thinking in just adding a comment like:
```
# Usage:
#     Update all languages: ./update.sh
#     Update a specific language: ./update.sh de
```

**Pull request merged**